src/python/qeqiskit/noise/basic.py

```diff
@@ -38,3 +44,118 @@ def get_qiskit_noise_model(
     coupling_map = noisy_device.configuration().coupling_map
 
     return noise_model, CircuitConnectivity(coupling_map)
+
+def create_amplitude_damping_noise(T_1, t_step=10e-9):
+    """ Creates an amplitude damping noise model
+    
+    Args:
+        T_1 (float) : Relaxation time
+        t_step (float) : Discretized time step over which the relaxation occurs over
+    
+    Returns:
+        qiskit.providers.aer.noise.NoiseModel
+    """
+
+    gamma = (1 - pow(np.e, - 1/T_1*t_step))
+    error = amplitude_damping_error(gamma)
+    gate_error = error.tensor(error)
+
+    noise_model = NoiseModel()
+    noise_model.add_all_qubit_quantum_error(error, ['id', 'u3'])
+    noise_model.add_all_qubit_quantum_error(gate_error, ['cx'])
+    return noise_model
+
+def create_phase_damping_noise(T_2, t_step=10e-9):
+    """ Creates a dephasing noise model
+    
+    Args:
+        T_2 (float) : dephasing time
+        t_step (float) : Discretized time step over which the relaxation occurs over
+    
+    Returns:
+        qiskit.providers.aer.noise.NoiseModel
+    """
+    gamma = (1 - pow(np.e, - 1/T_2*t_step))
+    error = phase_damping_error(gamma)
+    gate_error = error.tensor(error)
+
+    noise_model = NoiseModel()
+    noise_model.add_all_qubit_quantum_error(error, ['id', 'u3'])
+    noise_model.add_all_qubit_quantum_error(gate_error, ['cx'])
+    return noise_model
+
+def create_phase_and_amplitude_damping_error(T_1, T_2, t_step=10e-9):
+    """ Creates a noise model that does both phase and amplitude damping
+    
+    Args:
+        T_1 (float) : Relaxation time
+        T_2 (float) : dephasing time
+        t_step (float) : Discretized time step over which the relaxation occurs over
+    
+    Returns:
+        qiskit.providers.aer.noise.NoiseModel
+    """
+
+    param_amp = (1 - pow(np.e, - 1/T_1*t_step))
+    param_phase = (1 - pow(np.e, - 1/T_2*t_step))
+    error = phase_amplitude_damping_error(param_amp, param_phase)
+    gate_error = error.tensor(error)
+
+    noise_model = NoiseModel()
+    noise_model.add_all_qubit_quantum_error(error, ['id', 'u3'])
+    noise_model.add_all_qubit_quantum_error(gate_error, ['cx'])
+    return noise_model
+
+def create_pta_channel(T_1, T_2, t_step=10e-9):
+    """ Creates a noise model that does both phase and amplitude damping but in the
+        Pauli Twirling Approximation discussed the following reference 
+        https://arxiv.org/pdf/1305.2021.pdf
+
+    
+    Args:
+        T_1 (float) : Relaxation time
+        T_2 (float) : dephasing time
+        t_step (float) : Discretized time step over which the relaxation occurs over
+    
```

Comment:
Another question: @akataba can you put in the units for the `T_1`, `T_2`, and `t_step` here and for the other functions?

Reply:
Done